src/precompiles/ArbMathStats.sol

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro-contracts/blob/main/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity >=0.4.21 <0.9.0;
+
+/// @title Statistics
+/// @notice This contract is used to compute statistics.
+/// This custom contract will set on 0x000000000000000000000000000000000000011b since we set it in precompile.go.
+interface ArbMathStats {
+    /**
+     * @notice Returns the standard deviation of input
+     * @return stddev floating point number represented as an int, with the given number of decimals taken
+     */
+    function stdDev(int32[] memory input, uint8 returnDecimals) external returns (int256);
```

Comment:
This can probably be a view function right? Doesn't modify state or write anything special to DA...etc.